src/renderer/components/hotbar/hotbar-icon.tsx

```diff
@@ -75,33 +70,25 @@ export const HotbarIcon = observer(({ menuItems = [], size = 40, tooltip, ...pro
   };
 
   return (
-    <div className={cssNames("HotbarIcon flex", className, { disabled, contextMenuAvailable: menuItems.length > 0 })}>
+    <div className={cssNames(styles.HotbarIcon, className, { [styles.contextMenuAvailable]: menuItems.length > 0 })}>
       {tooltip && <Tooltip targetId={id}>{tooltip}</Tooltip>}
-      <div
+      <Avatar
+        {...rest}
         id={id}
-        onClick={(event) => {
-          if (!disabled) {
-            onClick?.(event);
-          }
-        }}
+        title={title}
+        colorHash={`${title}-${source}`}
+        className={cssNames({ [styles.active]: active })}
+        disabled={disabled}
+        size={size}
+        src={src}
+        onClick={(event) => !disabled && onClick?.(event)}
       >
-        <Avatar
-          {...rest}
-          title={title}
-          colorHash={`${title}-${source}`}
-          className={cssNames(active ? "active" : "default", { interactive: !!onClick })}
-          width={size}
-          height={size}
-          src={src}
-        >
-          {material && <Icon className="materialIcon" material={material} />}
-        </Avatar>
-        {children}
-      </div>
+        {material && <Icon material={material} />}
```

Comment:
What if `material` is an svg?

Reply:
Currently entities support only string icon notation and icons being used for General catalog entities, like here https://github.com/lensapp/lens/blob/master/src/common/catalog-entities/general.ts#L60

Reply:
What about KubernetesCluster entities https://github.com/lensapp/lens/blob/master/src/common/catalog-entities/kubernetes-cluster.ts#L49-L54?

Reply:
This line `src?: string` relates to image URL and gets passed to `<Avatar src={src}/>` directly.